h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -288,6 +308,22 @@ function SidebarInjector(chromeTabs, dependencies) {
   function injectScript(tabId, path) {
     return executeScriptFn(tabId, {file: path});
   }
+
+  /**
+   * Inject configuration information for the Hypothesis application
+   * into the page as JSON data via a <meta> tag.
+   *
+   * A <meta> tag is used because that makes it available to JS content
+   * running in isolated worlds.
+   */
+  function injectConfig(tabId, config) {
```

Comment:
I know they're not the same settings. I'm merely making the point that we probably should use the same basic mechanism unless we have a good reason to do otherwise.